neo/Consensus/ConsensusContext.cs

```diff
@@ -24,19 +24,22 @@ internal class ConsensusContext : IConsensusContext
         public ECPoint[] Validators { get; set; }
         public int MyIndex { get; set; }
         public uint PrimaryIndex { get; set; }
+        public ConsensusPayload[] ChangeViewPayloads { get; set; }
         public uint Timestamp { get; set; }
         public ulong Nonce { get; set; }
         public UInt160 NextConsensus { get; set; }
         public UInt256[] TransactionHashes { get; set; }
         public Dictionary<UInt256, Transaction> Transactions { get; set; }
         public UInt256[] Preparations { get; set; }
```

Comment:
Since we have `PreparationPayloads`, we can remove `Preparations`.

Reply:
Done.